packages/grid/src/styled/header.ts

```diff
@@ -69,6 +69,14 @@ export const HeaderCellContent = styled.div<IHeaderCellContent>`
   ${getHeaderCellContentStyles}
 `;
 
+export const SortIconWrapper = styled.div`
```

Comment:
Looks like this is unused. But i suppose you need it because if you have a lot of text in column, sort icon will shrink.
So you need flex-shrink: 0
![image](https://user-images.githubusercontent.com/17886232/77740861-a2be5800-701c-11ea-95b2-41508b226174.png)